packages/console-types/src/types/catalog/item.ts

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2025 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import type { FromSchema, JSONSchema } from 'json-schema-to-ts'
+
+import {
+  catalogComingSoonSchema,
+  catalogDescriptionSchema,
+  catalogDocumentationSchema,
+  catalogImageUrlSchema,
+  catalogIsLatestSchema,
+  catalogIsVersioningSupportedSchema,
+  catalogItemIdSchema,
+  catalogNameSchema,
+  catalogProviderIdSchema,
+  catalogPublishOnMiaDocumentationSchema,
+  catalogReleaseDateSchema,
+  catalogReleaseStageSchema,
+  catalogRepositoryUrlSchema,
+  catalogResourcesSchema,
+  catalogSupportedByImageUrlSchema,
+  catalogSupportedBySchema,
+  catalogTenantIdSchema,
+  catalogTypeSchema,
+  catalogSemverVersionSchema,
+  catalogVisibilitySchema,
+} from './commons'
+
+export const catalogItemSchema = {
```

Comment:
Why do we expose the schema? I have a doubt. 
Does the user need it? Shouldn't it be inside the specific microservices?

Reply:
Same for every schema added in this pr